packages/features/ee/organizations/components/CreateANewOrganizationForm.tsx

```diff
@@ -74,24 +79,27 @@ export const CreateANewOrganizationForm = ({ slug }: { slug?: string }) => {
     },
   });
 
+  const isAdmin = session.data?.user.role === UserPermissionRole.ADMIN;
+  const isImpersonated = session.data?.user.impersonatedBy;
+
```

Comment:
@sean-brydon how would you rewrite this for admins who are impersonating normal users?